lib/beaker/perf.rb

```diff
@@ -46,6 +46,10 @@ def setup_perf_on_host(host)
         @logger.perf_output("Creating symlink from /etc/sysstat/sysstat.cron to /etc/cron.d")
         host.exec(Command.new('ln -s /etc/sysstat/sysstat.cron /etc/cron.d'),:acceptable_exit_codes => [0,1])
       end
+      if @options[:collect_perf_data] =~ /aggressive/
+        @logger.perf_output("Enabling aggressive sysstat polling")
+        host.exec(Command.new('sed -i s/5-55\\\/10/*/ /etc/cron.d/sysstat'))
```

Comment:
This section needs to be modified; crontab format differs between OS releases (Debian and CentOS, at least).